test/e2e_node/container.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"errors"
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+//One pod one container
+type ConformanceContainer struct {
+	Container api.Container
+	Client    *client.Client
+	Status    string
+}
+
+func NewConformanceContainer(cl *client.Client, container api.Container) ConformanceContainer {
+	return ConformanceContainer{
+		Container: container,
+		Client:    cl,
+	}
+}
+
+func (cc *ConformanceContainer) Create() error {
+	pod := &api.Pod{
+		ObjectMeta: api.ObjectMeta{
+			//Same with the container name
+			Name:      cc.Container.Name,
+			Namespace: api.NamespaceDefault,
+		},
+		Spec: api.PodSpec{
+			NodeName:      *nodeName,
+			RestartPolicy: api.RestartPolicyNever,
+			Containers: []api.Container{
+				cc.Container,
+			},
+		},
+	}
+
+	_, err := cc.Client.Pods(api.NamespaceDefault).Create(pod)
+	return err
+}
+
+func (cc *ConformanceContainer) IsStarted() (bool, error) {
```

Comment:
Consider returning either a bool or error since they are redundant.  Your function either returns false, error or true, nil; and the caller doesn't need both.  I think just the bool is best.